src/Plutarch/Multivalidator.hs

```diff
@@ -44,28 +44,28 @@ spend = phoistAcyclic $
         txInfoF <- pletFieldsC @'["inputs", "wdrl"] ctxF.txInfo
         ownInput <- pletC $ ptryOwnInput # txInfoF.inputs # ownRef
         ownInputF <- pletFieldsC @'["address"] ownInput
-        PScriptCredential ownInputScriptCred <- pmatchC (pfield @"credential" # ownInputF.address)
+        PDJust (pfromData . (pfield @"_0" #) -> ownInputScriptCred) <- pmatchC (pfield @"stakingCredential" # ownInputF.address)
```

Comment:
It should be `payment_credential`

Reply:
https://github.com/Anastasia-Labs/aiken-trie/blob/c237d94f85ed5077eb2035ebf0275923940bf6b8/validators/trie.ak#L19